### ISSUE TITLE
finetune_on_pregenerate Loss.backwards() throw an error 

### ISSUE THREAD

**Issue by sophiapeng0426** [huggingface/pytorch-transformers#831]
In finetune_on_pregenerated.py, loss are tuples and thus loss.backward() is not going to work.

Original:
loss = model(input_ids, segment_ids, input_mask, lm_label_ids, is_next)

Update:
loss, _ , _ = model(input_ids, segment_ids, input_mask, lm_label_ids, is_next)

Is this fix correct?

**Comment by thomwolf** [comment]
Yes this example should have been updated more with #797.
